packages/eslint-config-4catalyzer-typescript/index.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  extends: 'plugin:@typescript-eslint/eslint-recommended',
```

Comment:
Turns out nesting overrides from other modules is... not great. So this needs to go at top level.